project/frontend/src/containers/CreateCompetition/CreateCompetition.js

```diff
@@ -1,10 +1,23 @@
-import React from "react";
+import React, { useState } from "react";
 import classes from "./CreateCompetition.module.css";
 import { Link } from "react-router-dom";
 import LinkButton from "../../components/UI/LinkButton/LinkButton";
+import ParticipantCard from "../../components/ParticipantCard/ParticipantCard";
 import { RiAddCircleLine } from "react-icons/ri";
 
 const CreateCompetition = (props) => {
+  let [participant, setParticipant] = useState("");
```

Comment:
The standard way is to use `const` for the useState hook rather than `let`

Reply:
Fixed here ea103173fd549aba5f7759bc1e64138595d91c41